libs/wire-api-federation/src/Wire/API/Federation/API/Brig.hs

```diff
@@ -61,19 +62,19 @@ data Api routes = Api
       routes
         :- "federation"
         :> "claim-prekey"
-        :> ReqBody '[JSON] (UserId, ClientId)
+        :> ReqBody '[JSON] (LegalholdProtectee, UserId, ClientId)
         :> Post '[JSON] (Maybe ClientPrekey),
     claimPrekeyBundle ::
       routes
         :- "federation"
         :> "claim-prekey-bundle"
-        :> ReqBody '[JSON] UserId
+        :> ReqBody '[JSON] (LegalholdProtectee, UserId)
         :> Post '[JSON] PrekeyBundle,
     claimMultiPrekeyBundle ::
       routes
         :- "federation"
         :> "claim-multi-prekey-bundle"
-        :> ReqBody '[JSON] UserClients
+        :> ReqBody '[JSON] (LegalholdProtectee, UserClients)
```

Comment:
I don't think this can work.  We need to get `ZUser` from the header, the request body cannot be expected to change, these are external end-points.

Reply:
Ah, get it.  this is between federated instances.  But I still don't know if we can just change it like this.  We shouldn't really touch federation here, it's not supposed to work yet.  I'll try to surgically remove this again.